packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -68,10 +68,8 @@
         class="indirect-link-list-toggle"
         appearance="raw"
         @click="toggleLinkListCollapsed"
-      >
-        <span v-text="collapseButtonTitle" />
-        <oc-icon :name="collapseButtonIcon" fill-type="line" />
-      </oc-button>
+        v-text="collapseButtonTitle"
```

Comment:
Please leave this as it was, it's considered bad practice to use `v-text` on components since it can break the component's content. Your IDE linter should also complain 🤔

Reply:
does not

Reply:
Then something is misconfigured. `eslint` rightfully gives a warning.